backend/src/main/java/se/chalmers/dat265/group1/storage/repository/GenericRepository.java

```diff
@@ -67,15 +68,15 @@ public GenericRepository(Class<T> type, DBInterface dbInterface){
 
         ResultSet rs =  this.dbInterface.executeQuerry(sb.toString());
         try {
-            return this.getMatchJSON(rs); //converts resulting set to model instance
+            return this.getMatchJSON(rs); //converts resulting set to se.chalmers.dat265.group1.model instance
         } catch (SQLException e) {
-            e.printStackTrace();
+            log.error("getObjects", e);
             return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)